README.md

```diff
@@ -126,6 +126,7 @@ Keep-Alive: timeout=5
 | tags                      | `UNLEASH_TAGS`                   | undefined       |    no    | Used to filter features by using tags set for features. Format should be `tagName:tagValue,tagName2:tagValue2`                                                                                                                                                                                                                                                                                                                                                           |
 | clientKeysHeaderName      | `CLIENT_KEY_HEADER_NAME`         | "authorization" |    no    | The name of the HTTP header to use for client keys. Incoming requests must set the value of this header to one of the Proxy's `clientKeys` to be authorized successfully.                                                                                                                                                                                                                                                                                                |
 | enableOAS                 | `ENABLE_OAS`                     | `false`         |    no    | Set to `true` to expose the proxy's OpenAPI spec at `/docs/openapi.json` and an interactive OpenAPI UI at `/docs/openapi`. Read more in the [OpenAPI section](#openapi).                                                                                                                                                                                                                                                                                                 |
+| enableAllEndpoint         | `ENABLE_ALL_ENDPOINT`            | `false`         |    no    | Set to `true` to expose the proxy's `/all` endpoint tat will return all the feature toggles.                                                                                                                                                                                                                                                                                                    |
```

Comment:
What is the full path to the new endpoint? Isn't it `/proxy/all`? If so, I think we should use the full path. There's also a little typo, but I think we should create a new section of the readme explaining that endpoint. Something like this:

```md
## Returning enabled **and** disabled toggles

By default, the proxy only returns enabled toggles. However, in certain use cases, you might want it to return **all** toggles, regardless of whether they're enabled or disabled. The `/proxy/all` endpoint does this.

Because returning all toggles regardless of their state is a potential security vulnerability, the endpoint has to be explicitly enabled. To enable it, set the `enableAllEndpoint` configuration option or the `ENABLE_ALL_ENDPOINT` environment variable to `true`.
```

With that, we could change it to something like this:

```suggestion
| enableAllEndpoint         | `ENABLE_ALL_ENDPOINT`            | `false`         |    no    | Set to `true` to expose the `/proxy/all` endpoint.  Refer to the [section on returning all toggles](#return-enabled-and-disabled-toggles) for more info.                                                                                                                                                                                                                                                                                                   |
```

What do you think?

Reply:
I like it  :+1: